include/image.mk

```diff
@@ -40,6 +40,10 @@ KDIR=$(KERNEL_BUILD_DIR)
 KDIR_TMP=$(KDIR)/tmp
 DTS_DIR:=$(LINUX_DIR)/arch/$(LINUX_KARCH)/boot/dts
 
+# If there is no explicit EXTRA_IMAGE_NAME use the unquoted one from the .config
+ifeq ($(EXTRA_IMAGE_NAME),)
```

Comment:
The usage of `ifeq` as well as your description is correct which reflects what the comment says and what the intended behavior is. :) That is: If there is no `EXTRA_IMAGE_NAME` yet, use the one from the .config.